src/dft/backends/cufft/backward.cpp

```diff
@@ -85,6 +100,15 @@ ONEMKL_EXPORT void compute_backward(descriptor_type &desc,
     auto commit = detail::checked_get_commit(desc);
     auto queue = commit->get_queue();
     auto plan = detail::get_bwd_plan(commit);
+    auto offsets = detail::get_offsets(commit);
+
+    if constexpr (std::is_floating_point_v<fwd<descriptor_type>>) {
+        if (offsets[1] % 2 != 0) {
+            throw oneapi::mkl::unimplemented(
+                "DFT", "compute_forward(desc, inout)",
```

Comment:
The function arguments in this string and others are incorrect. can you utilize `func_name` from the host task?

Reply:
yes, fixed